src/app/web-app.component.ts

```diff
@@ -45,6 +48,8 @@ export class WebAppComponent implements OnInit {
    * @param {ThemeStorageService} themeStorageService Theme Storage Service.
    * @param {MatSnackBar} snackBar Material Snackbar for notifications.
    * @param {AlertService} alertService Alert Service.
+   * @param {KeyboardShortcutsService} keyboardShortcuts KeyboardS Shortcuts Service.
+   * @param {SearchService} searchSevice Searchbar Visibility variable service
```

Comment:
Make the comment same as you have put in toolbar component to ensure consistency. Also, authentication service hasn't been mentioned in the comment.